src/components/application_manager/test/mobile_message_handler_test.cc

```diff
@@ -233,13 +233,15 @@ class MobileMessageHandlerTest : public testing::Test {
                                                             data,
                                                             bin_dat);
     // Act
-    RawMessage* result_message =
-        MobileMessageHandler::HandleOutgoingMessageProtocol(message_to_send);
+    std::shared_ptr<RawMessage> result_message(
```

Comment:
@SNiukalov Can't see any sharing here. What is the reason to create shared_ptr?

Reply:
@AByzhynar 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/41/commits/be4a738c2392d04c4fd0cd9a2dee39ce76b94c6e